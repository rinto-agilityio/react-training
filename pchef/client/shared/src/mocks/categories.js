export const categories = [
  {
    id: 'chicken',
    name: 'Chicken',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Heathy',
    name: 'Heathy',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Easy Breakast',
    name: 'Easy Breakast',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Dessert',
    name: 'Dessert',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Fish',
    name: 'Fish',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Pizza',
    name: 'Pizza',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Cake',
    name: 'Cake',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Deserts',
    name: 'Deserts',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Salad',
    name: 'Salad',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Kids',
    name: 'Kids',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Meat',
    name: 'Meat',
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
  {
    id: 'Easy Lunch',
    name: null,
    imgUrl:
      'https://realhousemoms.com/wp-content/uploads/Honey-Butter-Chicken-IG.jpg',
  },
]
