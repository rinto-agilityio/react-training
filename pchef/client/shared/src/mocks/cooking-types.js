export const cookingTypes = [
  {
    id: 'baking',
    name: 'Baking',
  },
  {
    id: 'roasting',
    name: 'Roasting',
  },
  {
    id: 'frying',
    name: 'Frying',
  },
]
