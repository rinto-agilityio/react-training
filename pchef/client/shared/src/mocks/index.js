// mock categories
export const category = {
  name: 'Heathy',
  imgUrl:
    'https://images.pexels.com/photos/461198/pexels-photo-461198.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
}
