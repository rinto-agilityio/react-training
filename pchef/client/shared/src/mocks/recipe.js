export const recipe = {
  description: '2lb pork neck bones, 2lb beef bones, 1lb oxtail',
  title: 'Bun Bo Hue Recipe',
  imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
  votes: [1],
  steps: [
    {
      step: 1,
      description: 'Step 1 description',
      title: 'Step 1 title',
      subTitle: 'Step 1 sub title',
      imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    },
    {
      step: 2,
      description: 'Step 2 description',
      title: 'Step 2 title',
      subTitle: 'Step 2 sub title',
      imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    },
    {
      step: 3,
      description: 'Step 3 description',
      title: 'Step 3 title',
      subTitle: 'Step 3 sub title',
      imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    },
    {
      step: 4,
      description: 'Step 4 description',
      title: 'Step 4 title',
      subTitle: 'Step 4 sub title',
      imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    },
    {
      step: 5,
      description: 'Step 5 description',
      title: 'Step 5 title',
      subTitle: 'Step 5 sub title',
      imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    },
    {
      step: 6,
      description: 'Step 6 description',
      title: 'Step 6 title',
      subTitle: 'Step 6 sub title',
      imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    },
  ],
}
