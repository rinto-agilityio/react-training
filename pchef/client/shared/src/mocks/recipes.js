export const recipes = [
  {
    id: 0,
    title: 'Bun Bo Hue Recipe',
    subTitle: 'Traditional food',
    description: '2lb pork neck bones, 2lb beef bones, 1lb oxtail',
    imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
    votes: [1],
    views: 1,
    steps: [
      {
        step: 1,
        description: 'Step 1 description',
        title: 'Step 1 title',
        subTitle: 'Step 1 sub title',
        imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
      },
      {
        step: 2,
        description: 'Step 2 description',
        title: 'Step 2 title',
        subTitle: 'Step 2 sub title',
        imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
      },
      {
        step: 3,
        description: 'Step 3 description',
        title: 'Step 3 title',
        subTitle: 'Step 3 sub title',
        imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
      },
      {
        step: 4,
        description: 'Step 4 description',
        title: 'Step 4 title',
        subTitle: 'Step 4 sub title',
        imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
      },
      {
        step: 5,
        description: 'Step 5 description',
        title: 'Step 5 title',
        subTitle: 'Step 5 sub title',
        imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
      },
      {
        step: 6,
        description: 'Step 6 description',
        title: 'Step 6 title',
        subTitle: 'Step 6 sub title',
        imgUrl: 'https://vcdn-ngoisao.vnecdn.net/2018/01/19/1-8515-1516314392.jpg',
      },
    ],
  },
  {
    id: 1,
    title: 'Crispy Fried Chicken',
    imgUrl:
      'https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg',

    description: 'Description for Crispy Fried Chicken',
  },
  {
    id: 2,
    title: 'Crispy Fried Chicken',
    imgUrl:
      'https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg',

    description: 'Description for Crispy Fried Chicken',
  },
  {
    id: 3,
    title: 'Crispy Fried Chicken',
    imgUrl:
      'https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg',

    description: 'Description for Crispy Fried Chicken',
  },
]
