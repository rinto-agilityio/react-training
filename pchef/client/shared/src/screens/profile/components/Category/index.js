// Libs
import React from 'react'
import { Text, View } from 'react-native'

// Styles
import styles from './styles'

// Components
import Image from '../../../../components/Image'

type Props = {
  category: {
    name: string,
    imgUrl: string,
  },
}

const Category = ({ category }: Props) => {
  const { name, imgUrl } = category

  return (
    <View style={styles.wrapper}>
      <Text style={styles.title}>{name}</Text>
      <Image url={imgUrl} customImageStyle={styles.image} />
    </View>
  )
}

export default Category
