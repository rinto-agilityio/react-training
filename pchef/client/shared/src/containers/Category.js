import { compose } from 'react-apollo'

// import graphQL query + component
import { getCategory } from '../graphql/category'
import { userToggleRecipe } from '../graphql/recipe'
import { getRecipes } from '../graphql/recipe'

// import component
import CategoryScreen from '../screens/category'

export default compose(getCategory, userToggleRecipe, getRecipes)(CategoryScreen)
