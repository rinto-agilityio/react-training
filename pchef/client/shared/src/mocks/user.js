import { categories } from './categories'
import { recipes } from './recipes'

const favoriteRecipeId: Array<string> = recipes.map(item => item.id)
export const user = {
  id: '1',
  name: 'Alex',
  email: 'alex@gmail.com',
  avatar:
    'https://cdn4.iconfinder.com/data/icons/avatars-21/512/avatar-circle-human-female-black-7-512.png',
  favoriteRecipe: favoriteRecipeId,
  followCategory: categories,
}
