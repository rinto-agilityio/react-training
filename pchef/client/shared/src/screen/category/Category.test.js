// Libs
import React from 'react'
import renderer from 'react-test-renderer'

// Mocks
import { category, recipes } from '../../mocks'

// Components
import Category from '.'

const props = {
  category,
  recipes,
}
describe('Screen', () => {
  describe('Category', () => {
    let component

    beforeEach(() => {
      beforeEach(() => {
        component = shallow(<Category {...props} />)
      })
    })

    it('should render Category screen', () => {
      const renderComponent = renderer.create(<Category {...props} />)
      expect(renderComponent).toMatchSnapshot()
    })

    it('should render Recipe component', () => {
      expect(component.find('Recipe')).toHaveLength(recipes.length)
    })

    it('should render Header component', () => {
      expect(component.find('Header')).toHaveLength(1)
    })
  })
})
