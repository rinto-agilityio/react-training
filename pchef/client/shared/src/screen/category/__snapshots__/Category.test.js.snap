// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen Category should render Category screen 1`] = `
Array [
  <div
    className="css-view-1dbjc4n r-display-6koalj r-height-1kl1mp1"
  >
    <div
      className="css-view-1dbjc4n r-height-1pi2tsx r-width-13qz1uu"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxPack": "center",
          "WebkitJustifyContent": "center",
          "alignItems": "center",
          "justifyContent": "center",
          "msFlexAlign": "center",
          "msFlexPack": "center",
        }
      }
    >
      <div
        className="css-view-1dbjc4n r-bottom-1p0dtai r-flexBasis-1mlwlqe r-left-1d2f490 r-overflow-1udh08x r-position-u8s1d r-right-zchlnj r-top-ipm5af r-zIndex-1wyyakw"
      >
        <div
          className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
          style={
            Object {
              "backgroundImage": "url(\\"https://images.pexels.com/photos/461198/pexels-photo-461198.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500\\")",
            }
          }
        />
        <img
          alt=""
          className="css-accessibilityImage-9pa8cd"
          draggable={false}
          src="https://images.pexels.com/photos/461198/pexels-photo-461198.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500"
        />
      </div>
      <div
        className="css-text-76zvg2 r-color-jwli3a r-fontSize-1i10wst r-fontWeight-1kfrs79 r-marginBottom-d0pm55"
        dir="auto"
      >
        Heathy
      </div>
      <div
        className="css-view-1dbjc4n r-backgroundColor-1niwhzg"
        style={
          Object {
            "WebkitAlignItems": "center",
            "WebkitBoxAlign": "center",
            "WebkitBoxPack": "center",
            "WebkitJustifyContent": "center",
            "alignItems": "center",
            "borderBottomColor": "rgba(255,255,255,1.00)",
            "borderBottomLeftRadius": "20px",
            "borderBottomRightRadius": "20px",
            "borderBottomStyle": "solid",
            "borderBottomWidth": "2px",
            "borderLeftColor": "rgba(255,255,255,1.00)",
            "borderLeftStyle": "solid",
            "borderLeftWidth": "2px",
            "borderRightColor": "rgba(255,255,255,1.00)",
            "borderRightStyle": "solid",
            "borderRightWidth": "2px",
            "borderTopColor": "rgba(255,255,255,1.00)",
            "borderTopLeftRadius": "20px",
            "borderTopRightRadius": "20px",
            "borderTopStyle": "solid",
            "borderTopWidth": "2px",
            "boxShadow": "0px 0.75px 1.5px rgba(0,0,0,0.24)",
            "justifyContent": "center",
            "minWidth": "64px",
            "msFlexAlign": "center",
            "msFlexPack": "center",
            "paddingBottom": "5px",
            "paddingLeft": "20px",
            "paddingRight": "20px",
            "paddingTop": "5px",
          }
        }
      >
        <div
          className="css-cursor-18t94o4 css-view-1dbjc4n r-backgroundColor-1niwhzg r-cursor-1loqt21 r-overflow-1udh08x r-touchAction-1otgn73 r-userSelect-lrvibr"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          role="button"
          style={
            Object {
              "borderBottomLeftRadius": "4px",
              "borderBottomRightRadius": "4px",
              "borderTopLeftRadius": "4px",
              "borderTopRightRadius": "4px",
            }
          }
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz r-justifyContent-1777fci"
          >
            <div
              className="css-text-76zvg2 css-textOneLine-bfa6kz r-letterSpacing-ud0q2t r-marginHorizontal-1ydw1k6 r-marginVertical-712mwr r-textAlign-q4m81j"
              dir="auto"
              style={
                Object {
                  "color": "rgba(0,0,0,1.00)",
                  "direction": "ltr",
                  "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                }
              }
            >
              <span
                className="css-text-76zvg2 css-textHasAncestor-16my406 r-color-jwli3a r-fontSize-1b43r93 r-fontWeight-1kfrs79 r-textTransform-tsynxw"
                dir="auto"
                style={
                  Object {
                    "direction": "ltr",
                    "fontFamily": "Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                    "textAlign": "left",
                  }
                }
              >
                Following
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-4rzz9f r-display-6koalj r-flexDirection-18u37iz r-justifyContent-1777fci r-padding-1sp51qo"
  >
    <div
      className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
      style={
        Object {
          "marginRight": "30px",
        }
      }
    >
      <div
        className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
        data-focusable={true}
        disabled={false}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        role="button"
        tabIndex="0"
      >
        <span
          className="css-hitSlop-mjp8i1"
          style={
            Object {
              "bottom": "-6px",
              "left": "-6px",
              "right": "-6px",
              "top": "-6px",
            }
          }
        />
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-text-76zvg2 r-backgroundColor-1niwhzg r-pointerEvents-633pao"
            dir="auto"
            focusable={false}
            role="img"
            style={
              Object {
                "color": "rgba(31,51,64,1.00)",
                "fontSize": "20px",
              }
            }
          >
            □
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-alignItems-1awozwy r-flexDirection-18u37iz"
    >
      <div
        className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
        data-focusable={true}
        disabled={false}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onKeyUp={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        role="button"
        tabIndex="0"
      >
        <span
          className="css-hitSlop-mjp8i1"
          style={
            Object {
              "bottom": "-6px",
              "left": "-6px",
              "right": "-6px",
              "top": "-6px",
            }
          }
        />
        <div
          className="css-view-1dbjc4n"
        >
          <div
            className="css-text-76zvg2 r-backgroundColor-1niwhzg r-pointerEvents-633pao"
            dir="auto"
            focusable={false}
            role="img"
            style={
              Object {
                "color": "rgba(197,197,197,1.00)",
                "fontSize": "20px",
              }
            }
          >
            □
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="css-view-1dbjc4n r-marginTop-1x0uki6 r-position-bnwqim"
  >
    <div
      className="css-view-1dbjc4n r-borderBottomWidth-qklmqi r-height-cqilzk r-marginBottom-117bsoe r-marginLeft-1n0xq6e r-marginRight-zso239 r-position-bnwqim"
    >
      <div
        className="css-text-76zvg2 r-fontSize-zz5t5d r-paddingBottom-1mi0q7o"
        dir="auto"
      >
        Crispy Fried Chicken
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-flexDirection-18u37iz r-flexWrap-1w6e6rj r-justifyContent-1h0z5md"
        style={
          Object {
            "WebkitBoxFlex": 0,
            "WebkitFlexGrow": 0,
            "flexGrow": 0,
            "msFlexPositive": 0,
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-height-erd55g r-overflow-1udh08x r-zIndex-417010"
            style={
              Object {
                "width": "300px",
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
              style={
                Object {
                  "backgroundImage": "url(\\"https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg\\")",
                }
              }
            />
            <img
              alt=""
              className="css-accessibilityImage-9pa8cd"
              draggable={false}
              src="https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg"
            />
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-flexDirection-eqz5dr r-flexWrap-1w6e6rj r-height-erd55g r-justifyContent-1wtj0ep r-marginBottom-15d164r r-marginLeft-1n0xq6e r-marginRight-zso239 r-paddingLeft-1vwolk8 r-width-11gcy1e"
          style={
            Object {
              "WebkitBoxFlex": 0,
              "WebkitFlexGrow": 0,
              "flexGrow": 0,
              "msFlexPositive": 0,
            }
          }
        >
          <div
            className="css-text-76zvg2 r-fontSize-1i10wst r-height-9njtsq r-lineHeight-waer2z r-overflow-1udh08x"
            dir="auto"
          >
            Description for Crispy Fried Chicken
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-bottom-1ebgqk7 r-flexDirection-18u37iz r-position-u8s1d r-right-r19jdo"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-76zvg2 r-backgroundColor-1niwhzg r-fontSize-zz5t5d r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "color": "rgba(128,128,128,1.00)",
                    }
                  }
                >
                  □
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-borderBottomWidth-qklmqi r-height-cqilzk r-marginBottom-117bsoe r-marginLeft-1n0xq6e r-marginRight-zso239 r-position-bnwqim"
    >
      <div
        className="css-text-76zvg2 r-fontSize-zz5t5d r-paddingBottom-1mi0q7o"
        dir="auto"
      >
        Crispy Fried Chicken
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-flexDirection-18u37iz r-flexWrap-1w6e6rj r-justifyContent-1h0z5md"
        style={
          Object {
            "WebkitBoxFlex": 0,
            "WebkitFlexGrow": 0,
            "flexGrow": 0,
            "msFlexPositive": 0,
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-height-erd55g r-overflow-1udh08x r-zIndex-417010"
            style={
              Object {
                "width": "300px",
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
              style={
                Object {
                  "backgroundImage": "url(\\"https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg\\")",
                }
              }
            />
            <img
              alt=""
              className="css-accessibilityImage-9pa8cd"
              draggable={false}
              src="https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg"
            />
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-flexDirection-eqz5dr r-flexWrap-1w6e6rj r-height-erd55g r-justifyContent-1wtj0ep r-marginBottom-15d164r r-marginLeft-1n0xq6e r-marginRight-zso239 r-paddingLeft-1vwolk8 r-width-11gcy1e"
          style={
            Object {
              "WebkitBoxFlex": 0,
              "WebkitFlexGrow": 0,
              "flexGrow": 0,
              "msFlexPositive": 0,
            }
          }
        >
          <div
            className="css-text-76zvg2 r-fontSize-1i10wst r-height-9njtsq r-lineHeight-waer2z r-overflow-1udh08x"
            dir="auto"
          >
            Description for Crispy Fried Chicken
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-bottom-1ebgqk7 r-flexDirection-18u37iz r-position-u8s1d r-right-r19jdo"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-76zvg2 r-backgroundColor-1niwhzg r-fontSize-zz5t5d r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "color": "rgba(128,128,128,1.00)",
                    }
                  }
                >
                  □
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="css-view-1dbjc4n r-borderBottomWidth-qklmqi r-height-cqilzk r-marginBottom-117bsoe r-marginLeft-1n0xq6e r-marginRight-zso239 r-position-bnwqim"
    >
      <div
        className="css-text-76zvg2 r-fontSize-zz5t5d r-paddingBottom-1mi0q7o"
        dir="auto"
      >
        Crispy Fried Chicken
      </div>
      <div
        className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-flexDirection-18u37iz r-flexWrap-1w6e6rj r-justifyContent-1h0z5md"
        style={
          Object {
            "WebkitBoxFlex": 0,
            "WebkitFlexGrow": 0,
            "flexGrow": 0,
            "msFlexPositive": 0,
          }
        }
      >
        <div
          className="css-view-1dbjc4n r-cursor-1loqt21 r-touchAction-1otgn73 r-transitionDuration-eafdt9 r-transitionProperty-1i6wzkk r-userSelect-lrvibr"
          data-focusable={true}
          disabled={false}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          tabIndex="0"
        >
          <div
            className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-height-erd55g r-overflow-1udh08x r-zIndex-417010"
            style={
              Object {
                "width": "300px",
              }
            }
          >
            <div
              className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
              style={
                Object {
                  "backgroundImage": "url(\\"https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg\\")",
                }
              }
            />
            <img
              alt=""
              className="css-accessibilityImage-9pa8cd"
              draggable={false}
              src="https://www.tasteofhome.com/wp-content/uploads/2018/01/Crispy-Fried-Chicken_EXPS_FRBZ19_6445_C01_31_3b.jpg"
            />
          </div>
        </div>
        <div
          className="css-view-1dbjc4n r-alignItems-1habvwh r-display-6koalj r-flexDirection-eqz5dr r-flexWrap-1w6e6rj r-height-erd55g r-justifyContent-1wtj0ep r-marginBottom-15d164r r-marginLeft-1n0xq6e r-marginRight-zso239 r-paddingLeft-1vwolk8 r-width-11gcy1e"
          style={
            Object {
              "WebkitBoxFlex": 0,
              "WebkitFlexGrow": 0,
              "flexGrow": 0,
              "msFlexPositive": 0,
            }
          }
        >
          <div
            className="css-text-76zvg2 r-fontSize-1i10wst r-height-9njtsq r-lineHeight-waer2z r-overflow-1udh08x"
            dir="auto"
          >
            Description for Crispy Fried Chicken
          </div>
          <div
            className="css-view-1dbjc4n r-alignItems-1awozwy r-bottom-1ebgqk7 r-flexDirection-18u37iz r-position-u8s1d r-right-r19jdo"
          >
            <div
              className="css-cursor-18t94o4 css-view-1dbjc4n r-alignItems-1awozwy r-backgroundColor-1niwhzg r-borderRadius-6ncur5 r-cursor-1loqt21 r-height-1r8g8re r-justifyContent-1777fci r-margin-5scogr r-overflow-bztko3 r-touchAction-1otgn73 r-userSelect-lrvibr r-width-1acpoxo"
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              tabIndex="0"
            >
              <span
                className="css-hitSlop-mjp8i1"
                style={
                  Object {
                    "bottom": "-6px",
                    "left": "-6px",
                    "right": "-6px",
                    "top": "-6px",
                  }
                }
              />
              <div
                className="css-view-1dbjc4n"
              >
                <div
                  className="css-text-76zvg2 r-backgroundColor-1niwhzg r-fontSize-zz5t5d r-pointerEvents-633pao"
                  dir="auto"
                  focusable={false}
                  role="img"
                  style={
                    Object {
                      "color": "rgba(128,128,128,1.00)",
                    }
                  }
                >
                  □
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
